tools/picard/picard_CollectSequencingArtifactsMetrics.xml

```diff
@@ -0,0 +1,53 @@
+<tool id='picard_artifact_metrics' name='Picard Collect Sequencing Artifact Metrics'>
+  <description>writes 5 tables containing Phred-scaled error rates substitution details and summaries</description>
+  <!--<requirements>
+    <requirement type="package">picard</requirement>
+  </requirements>-->
+  <command>
+    <![CDATA[
+    
+    ##set up input files
+
+    #set $reference_fasta_filename = "/Users/aerijman/Documents/projects/galaxy/tool-data/hg38/seq/hg38.fa"
+    
+    picard -Xmx4048m
+    CollectSequencingArtifactMetrics
+    I='${sam_bam}'
+    O='${output_name}'
+    R='/Users/aerijman/Documents/projects/galaxy/tool-data/hg38/seq/hg38.fa' 
+    CONTEXT_SIZE=${context_size}
+    CONTEXTS_TO_PRINT="${context_print}"
+    INCLUDE_DUPLICATES="${duplicates}"
+    MINIMUM_QUALITY_SCORE="${min_quality_score}";
+
+    ]]>
+  </command>
+
+  <stdio>
+    <exit_code range="1:"  level="fatal" />
+  </stdio>
+  
+  <inputs>
+    <!--<param name='output_name' type='text' label='prefix to name output files'>-->
+    <param name='sam_bam' type='data' label='Sam/Bam file'/>
+    <param name='reference_fasta_filename' type='data' label='Reference genome' help='last reference genome used is default'/>
+    <param name='context_size' value="1" type='integer' label='Context size' helper='number of bases on each side' help='default is 1'/>
+    <param name='output_name' type="text" label="give prefix for the ouput files"/>
+    <!--<repeat name="contexts" title="Contexts to print" min="0" help='only these will be printed in detailed output tables but all are considered for summary'>
+      <param name='contexts_to_print' value='1' type='integer' label='Include this context'/>
+    </repeat>-->
+    <param name="context_print" type="text" label="Contest to print" help="eg. AAA"/>    
+    <param name='db_snp' type='data' label='Database of SNPs' helper='to exclude regions around polymorphisms (VCF format)'/>
+    <param name='duplicates' type='boolean' label='Include duplicates' checked="false" truevalue="true" falsevalue="false" help="Assume False"/>
+    <param name='min_quality_score' value='30' type='integer' label='Minimum quality score'/>
+  </inputs>
+
+  <outputs>
+    <data name='pre_details' format="tabular"  from_work_dir="/Users/aerijman/Documents/projects/galaxy/testing_output_probando.pre_adapter_detail_metrics" label='pre adapter - detials'/>
```

Comment:
needs more outputs. tool produces 5 right?

Reply:
This was outdated...